sandbox/version-control-manager/src/main/java/com/vikas/method_parser/version_control_manager/details/fetcher/UrlDetailesFetcher.java

```diff
@@ -0,0 +1,60 @@
+package com.vikas.method_parser.version_control_manager.details.fetcher;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientResponse;
+import com.sun.jersey.api.client.WebResource;
+
+public class UrlDetailesFetcher {
+
+	//Calling client using URLConnection
+    public static void getRepo(URL orgUrl) throws IOException {
```

Comment:
Again wrong-naming, make the method generic.

Reply:
Also, the parameter also should be generic.

Reply:
This piece of code is just dummy. It will not be used so I will remove it.